config/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"reflect"
+
+	"github.com/coreos/ignition/config/types"
+	"github.com/go-yaml/yaml"
+)
+
+func ParseAsNewest(data []byte) (types.Config, error) {
+	return ParseAsV2_0_0(data)
+}
+
+func ParseAsOldest(data []byte) (types.Config, error) {
```

Comment:
Consuming this lib, I can't think of a case I'd prefer `ParseAsOldest` over the specific `ParseAsV2_0_0` or whatever it is? I'd prefer obvious breakage if `ParseV2_0_0` were removed over the min version quietly bumping up.